codility_training/lessons.lesson07.StackandQueues.StoneWall/eyabc.js

```diff
@@ -0,0 +1,13 @@
+function solution(H) {
+  const stack = [H[0]];
```

Comment:
```suggestion
  const stack = [H[0]];
  stack.getLast = () => stack[stack.length - 1]
```
이렇게 stack에 종속적인 함수를 만들 수 있음